crypto/err/openssl.txt

```diff
@@ -744,6 +744,7 @@ EVP_F_EVP_ENCRYPTUPDATE:167:EVP_EncryptUpdate
 EVP_F_EVP_MAC_CTRL:209:EVP_MAC_ctrl
 EVP_F_EVP_MAC_CTRL_STR:210:EVP_MAC_ctrl_str
 EVP_F_EVP_MAC_CTX_COPY:211:EVP_MAC_CTX_copy
+EVP_F_EVP_MAC_CTX_DUP:221:EVP_MAC_CTX_dup
```

Comment:
You can remove the `EVP_F_EVP_MAC_CTX_COPY` line...  probably even reuse its number.